R/sets.r

```diff
@@ -21,21 +26,27 @@
 #' union_all(first, second)
 #' setequal(mtcars, mtcars[32:1, ])
 #'
-#' # Handling of duplicates:
-#' a <- data.frame(column = c(1:10, 10))
-#' b <- data.frame(column = c(1:5, 5))
+#' # Note these functions also remove pre-existing duplicates in x or y:
```

Comment:
```suggestion
#' # Note the following 3 functions also remove pre-existing duplicates in `x` or `y`:
```

I thought "these functions" referred to all of the functions on this page, which of course isn't true